tests/pipelines/ner/test_suicide_attempt.py

```diff
@@ -0,0 +1,18 @@
+def test_covid(blank_nlp):
+    examples = [
+        ("J'ai vu le patient à cause d'une TS médicamenetuse.", "TS"),
+        ("J'ai vu le patient à cause d'une ts médicamenetuse.", ""),
+        ("J'ai vu le patient à cause d'une IMV.", "IMV"),
+        ("surface TS", ""),
```

Comment:
Can you add some tests for all the suicide_attempt_modality value and test those values ?
You can draw inspiration from https://github.com/aphp/edsnlp/blob/master/tests/pipelines/ner/test_cim10.py to set annotations on the snippets

Reply:
done!